contracts/Removal.sol

```diff
@@ -26,12 +26,14 @@ contract Removal is
   struct Vintage {
     address supplier;
     uint16 vintage;
+    bytes32 uniqueId;
     // todo: location
     // todo: methodology
     // todo: supplier name
   }
 
   mapping(uint256 => Vintage) private _vintages;
+  mapping(bytes32 => uint256) private _vintageTokenIdMap;
```

Comment:
```suggestion
  mapping(bytes32 => uint256) private _tokenIdToVintage;
```

Reply:
this is actually mapping from information about the vintage (but not from an actual vintage struct because that can't be a map key) TO the tokenId.. so I'll update the name to `_vintageToTokenIdMap`